.github/workflows/unittests.yml

```diff
@@ -15,11 +15,15 @@
      timeout-minutes: 30 
      strategy:
        matrix:
-         os: ['macos-latest', 'ubuntu-latest', 'windows-latest']
-         environment-file: [.ci/37.yaml, .ci/38.yaml, .ci/39.yaml]
+         os: [ubuntu-latest]
+         environment-file: [.ci/37.yaml, .ci/38.yaml, .ci/39.yaml, .ci/310.yaml]
          include:
-           - environment-file: .ci/39-DEV.yaml
+           - environment-file: .ci/310-DEV.yaml
              os: ubuntu-latest
```

Comment:
With the current setup, you can remove this and add `310-DEV.yaml` to the list above on line 19.

Reply:
good catch!